release-notes/unreleased/IGAPP-750-SiteHelpFull-Icons.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-750
+show_in_stores: true
+platforms:
+  - native
```

Comment:
@f1sh1918 this is not a valid platform. The only valid platforms are `ios`, `android` and `web`.